docs/source/docker/integration/docker_trigger_from_api.rst

```diff
@@ -0,0 +1,184 @@
+.. _integration-docker-trigger-from-api:
+
+Trigger a Docker Job from the API using a Remote Datasource
+===========================================================
+
+Introduction
+------------
+The Lightly workflow can be completely automated by using the Lightly Docker as
```

Comment:
It is still called "Lightly Docker" here. Why not call it "Lightly compute worker"? Or would you wait with that until we have decided on another newer name?

Reply:
Let's wait with the renaming and then use find to find all the spots. So far we mostly used the word "docker" in connection. So we just search for all "docker" words :)